optaweb-employee-rostering-backend/pom.xml

```diff
@@ -230,65 +235,102 @@
             </excludedTestClasses>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>net.revelc.code.formatter</groupId>
+          <artifactId>formatter-maven-plugin</artifactId>
+          <version>2.11.0</version>
+          <configuration>
+            <configFile>${format.directory}/eclipse-format.xml</configFile>
+            <lineEnding>LF</lineEnding>
+            <skip>${formatter.skip}</skip>
+          </configuration>
+          <executions>
+            <execution>
+              <goals>
+                <goal>${formatter.goal}</goal>
+              </goals>
+            </execution>
+          </executions>
+        </plugin>
+        <plugin>
+          <groupId>net.revelc.code</groupId>
+          <artifactId>impsort-maven-plugin</artifactId>
+          <version>1.4.1</version>
+          <configuration>
+            <groups>java.,javax.,org.,com.</groups>
+            <staticGroups>*</staticGroups>
+            <!-- keep in sync with the formatter-maven-plugin -->
+            <skip>${formatter.skip}</skip>
+            <removeUnused>true</removeUnused>
+          </configuration>
+          <executions>
+            <execution>
+              <goals>
+                <goal>${impsort.goal}</goal>
+              </goals>
+            </execution>
+          </executions>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-checkstyle-plugin</artifactId>
+          <version>3.1.0</version>
```

Comment:
I'd remove this, just like we removed it in core.

Reply:
It not removed in core; that the only reason why I kept it.
`optaplanner-build-parent/pom.xml`:
```
      <plugin>
        <artifactId>maven-checkstyle-plugin</artifactId>
      </plugin>
```

Reply:
Interesting, must have missed that one when I was removing the checkstyle config. Will finish it. :-)